taxi/views.py

```diff
@@ -52,3 +87,14 @@
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.edit.UpdateView):
+    model = Driver
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:driver-list")
```

Comment:
Code Style issue: Use `-` to split words in URL identification parameter `name`, not the `_`. So, it should be `driver-list` instead of `driver_list`.